src/views/static/history/history-view.styles.js

```diff
@@ -0,0 +1,176 @@
+import { css } from 'lit-element';
+
+export default css`
+  .construction {
+    text-align: center;
+    color: blue;
+    font-weight: bold;
+  }
+
+  .construction-sub {
+    text-align: center;
+    color: blue;
+  }
+
+  #history {
+    display: block;
+    background-image: url('src/assets/img/background_wellcome.jpg');
+    background-size: cover;
+    background-attachment: fixed;
+  }
+
+  .backgroundpanel {
+    display: none;
+    position: fixed;
+    width: 100%;
+    height: 100%;
+    background: rgba(0, 0, 0, 0.3);
+    z-index: 10;
+    top: 0;
+    left: 0;
+  }
+
+  .main-border {
+    margin: 0px 25%;
+    padding-top: 70px;
+    padding-bottom: 50px;
+  }
+  .main-content {
+    background-color: #fff;
+    padding: 14px 40px;
+    border-top: 8px solid #221f19;
+  }
+
+  .more {
```

Comment:
example of duplication: this css class is written 6 times (it should be only once)

Reply:
history-view.styles.js is deleted